packages/client/src/store/selectors/bookings/__tests__/bookings.test.ts

```diff
@@ -43,7 +43,7 @@ const setupBookingsTest = ({
   date,
   slotsByDay,
 }: {
-  category: Category | DeprecatedCategory;
+  category: Category[] | DeprecatedCategory[];
```

Comment:
To avoid confusion here, let's move this logic down to `updateCustomer` part 👇 
props: `category: Category | DeprecatedCategory`, and then, below when creating customer, `...getCusrtomerBase(saul), category: [category]`